### ISSUE TITLE
Copy markdown or copy rich text

### ISSUE THREAD

**Issue by simonff** [mesop-dev/mesop#923]
When an LLM generated markdown code (with bold text, lists, tables, etc), it's correctly rendered in mesop. But if I copy it, I get plain text with formatting lost.

It would be great to have the option to copy either the source markdown or the correct rich text. Or maybe to at least disable markdown rendering.

**Comment by richard-to** [comment]
Yes, we're planning to add a copy button (https://github.com/google/mesop/issues/275). That's next up on my todo list.

**Comment by richard-to** [comment]
Actually, I forgot the copy button is for the code snippets specifically and not the whole output.

To copy the whole output, the best option right now is a web component. Example here: https://github.com/google/mesop/tree/main/mesop/examples/web_component/copy_to_clipboard

The nice thing about this approach is that it allows you to place the copy button where you want it. Will need some adjustments for downstream usage though.
